### PR TITLE
Deploy and configure ENS with hardhat

### PR DESCRIPTION
**Pull request by jeremyjams**

### CODE REVIEW

File: deploy/1_deploy-ens.ts [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    await setReverseName(appRegistryAddress, 'apps.v5.iexec.eth');
+    await setReverseName(datasetRegistryAddress, 'datasets.v5.iexec.eth');
+    await setReverseName(workerpoolRegistryAddress, 'workerpools.v5.iexec.eth');
+
+    /**
+     * Register domain on ENS.
+     */
+    async function registerDomain(label: string, domain: string = '') {
+        const name = domain ? `${label}.${domain}` : `${label}`;
+        const labelHash = label ? labelhash(label) : '0x0';
```
Comment (zguesmi) [inline deploy/1_deploy-ens.ts:75]: Why not?
```suggestion
        const labelHash = label ? labelhash(label) : ethers.constants.HashZero;
```

Comment (jeremyjams) [reply deploy/1_deploy-ens.ts]: Both works, I can update yes, see https://github.com/iExecBlockchainComputing/PoCo/pull/93/commits/33ec55da8ffbce25a76cc9670b0ad87632195a86 !

**Review by zguesmi — APPROVED** [review]
:+1:

**Review by gfournieriExec — APPROVED** [review]
Perfect thanks

**Pull request merged**